pkg/bbgo/position.go

```diff
@@ -33,6 +33,22 @@ type Position struct {
 
 	ExchangeFeeRates map[types.ExchangeName]ExchangeFee `json:"exchangeFeeRates"`
 
+	// Futures data fields
+	Isolated               bool             		  `json:"isolated"`
+	Leverage               fixedpoint.Value           `json:"leverage"`
+	InitialMargin          fixedpoint.Value           `json:"initialMargin"`
+	MaintMargin            fixedpoint.Value           `json:"maintMargin"`
+	OpenOrderInitialMargin fixedpoint.Value           `json:"openOrderInitialMargin"`
+	PositionInitialMargin  fixedpoint.Value           `json:"positionInitialMargin"`
+	UnrealizedProfit       fixedpoint.Value           `json:"unrealizedProfit"`
+	EntryPrice             fixedpoint.Value           `json:"entryPrice"`
```

Comment:
EntryPrice is like Average Price?

Reply:
It's not explained specifically in API docs. But I think so, too.

FYI: 
 - [API Github](https://github.com/adshao/go-binance/blob/15a43411ea9cea6234d3c4b02a473d78f9dcdfaa/v2/futures/account_service.go)
 -  [API Docs](https://binance-docs.github.io/apidocs/futures/en/#position-information-v2-user_data)